docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -372,16 +373,16 @@ fields:
       show if: school.special_education_director.requestor_knows_address
       required: False
 under: |
-  Because ${student.name.first} is not yet in school, you need to send this letter to the special education director
+  Because ${ student.name.first } is not yet in school, you need to send this letter to the special education director
   for where you live. You can find out who your special education director is by searching [here](https://eecweb.eec.state.ma.us/ChildCareSearch/Special.aspx).
 ---
 id: special education coordinator information
 question: |
-  Special Education Coordinator
+  Special Education **Coordinator**
 fields:
   - "Do you know the name of the {special education coordinator} for ${ student.familiar() }?": knows_special_education_director_name
     datatype: yesnoradio
-  - Recipient Title: school.recipient.title
+  - Coordinator Title: school.special_education_coordinator.title
```

Comment:
I'm assuming you made this conditional in the template instead? That probably is clearer, just wanted to make sure

Reply:
Not used in any of the templates. Probably should be added. 

I made this issue because of how confusing this was: https://github.com/LemmaLegalConsulting/docassemble-StudentEvaluations/issues/35